ipaserver/install/certs.py

```diff
@@ -72,12 +72,20 @@ def install_pem_from_p12(p12_fname, p12_passwd, pem_fname):
                  "-passin", "file:" + pwd.name])
 
 
-def install_key_from_p12(p12_fname, p12_passwd, pem_fname):
+def install_key_from_p12(
+    p12_fname, p12_passwd, pem_fname, out_passwd_fname=None
```

Comment:
This indention style is uncommon and hard to read. I'd prefer if you indent it by 8 and keep ``):`` on line 76.

Reply:
Ok.